src/Castle.Core/DynamicProxy/IInterceptorBase.cs

```diff
@@ -0,0 +1,56 @@
+﻿
+namespace Castle.DynamicProxy
+{
+	using System;
+
+	public class ExceptionContext
```

Comment:
This should be `sealed` IMO (as there's no reason why someone outside the library should be able to instantiate this type), or even better, a `readonly struct` (as it's essentially a 2-tuple).

Also, documentation never hurts.